.env.development

```diff
@@ -4,8 +4,11 @@
 # VITE_API_URL=https://360Api.gordon.edu/
 
 # @TRAIN
- VITE_API_URL=https://360ApiTrain.gordon.edu/
+# VITE_API_URL=https://360ApiTrain.gordon.edu/
 
 # @LOCALHOST
 # VITE_API_URL=http://localhost:51626/
 
+# @Rec-IM
+VITE_API_URL=https://360apirecim.gordon.edu/
+
```

Comment:
you want to remember to revert your .env files. These files shouldn't change because not everyone will be using the 360apirecim server